openedx/stanford/cms/envs/devstack.py

```diff
@@ -1,6 +1,7 @@
 from cms.envs.devstack import *
 
 
+CMS_BASE = 'localhost:8001'
```

Comment:
We need to explicitly override this for devstack, else the port is incorrect (18010).